File: utils/VectorEnginePlugin.ref/pares/funciones/primAntiAliasingWidthsubPixelDelta.c
/* VectorEnginePlugin>>#antiAliasingWidth:subPixelDelta: */
EXPORT(sqInt)
primAntiAliasingWidthsubPixelDelta(void)
{
	double aFloat;
	double otherFloat;

	if (!((isFloatObject(stackValue(1)))
		 && (isFloatObject(stackValue(0))))) {
		return primitiveFailFor(PrimErrBadArgument);
	}
	aFloat = stackFloatValue(1);
	otherFloat = stackFloatValue(0);
	antiAliasingWidth = aFloat;
	subPixelDelta = otherFloat;
	auxAntiAliasingWidthScaledInverse = 127.0 / antiAliasingWidth;
	if (!(failed())) {
		pop(2);
	}
	return null;
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primAntiAliasingWidthsubPixelDelta.js
function primAntiAliasingWidthsubPixelDelta(argCount) {
    if (!(isFloatObject(stackValue(1)) && isFloatObject(stackValue(0)))) {
        primitiveFailFor(PrimErrBadArgument); return false;
    }
    antiAliasingWidth = stackFloatValue(1);
    subPixelDelta = stackFloatValue(0);
    auxAntiAliasingWidthScaledInverse = 127.0 / antiAliasingWidth;
    if (!failed()) pop(2);
    return !failed();
}
